src/backend/api/meals.js

```diff
@@ -1,12 +1,87 @@
+
 const express = require("express");
+const app = express();
 const router = express.Router();
 const knex = require("../database");
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    if (request.query.maxPrice) {
+      const maxPrice = Number(request.query.maxPrice);
+      const regularPrice = await knex("meal").where("Price", "<", maxPrice);
```

Comment:
just to make sure....in the exercise...the table is called `meals` and not `meal`

Reply:
`router.get("/", async (request, response) => {
  try {
    // knex syntax for selecting things. Look up the documentation for knex for further info
    const titles = await knex("**meals**").select("title");
    response.json(titles);
  } catch (error) {
    throw error;
  }
});
`

Reply:
you can test it in your browse: `http://localhost:3000/api/meals`